ios/Sources/TimetableFeature/Detail/TimetableDetailFeature.swift

```diff
@@ -0,0 +1,36 @@
+import ComposableArchitecture
+import Foundation
+import Model
+
+public struct TimetableDetailState: Equatable {
+    public var timetable: AnyTimetableItem
+    public var isSheetPresented: URL?
+
+    var isShowingSheet: Bool {
+        isSheetPresented != nil
+    }
+
+    public init(
+        timetable: AnyTimetableItem,
+        isSheetPresented: URL? = nil
+    ) {
+        self.timetable = timetable
+        self.isSheetPresented = isSheetPresented
+    }
+}
+
+public enum TimetableDetailAction {
+    case tapLink(URL)
+    case hideSheet
+}
+
+public let timetableDetailReducer = Reducer<TimetableDetailState, TimetableDetailAction, TimetableLoadedEnvironment> { state, action, _ in
```

Comment:
How about this? 👀  or create `TimetableDetailEnvironment` 🙏🏼 
I think the latter is better.

```suggestion
public let timetableDetailReducer = Reducer<TimetableDetailState, TimetableDetailAction, Void> { state, action, _ in
```